src/site/content/en/blog/yahoo-japan-news/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: How CLS optimizations increased Yahoo! JAPAN News's page views per session by 15%
+subhead: How to get your PWA to show up next to platform-specific apps in system-level sharing UIs
```

Comment:
```suggestion
subhead: >
  Optimizing CLS by 0.2 led to a 15% increase in page
  views per session, 13% longer session durations, and a 1.72
  percentage point decrease in bounce rate.
```

@mihajlija make sure to catch copy-paste errors like this when you have admin privileges! ;P